src/index.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+
+const theToDos = [
```

Comment:
- [ ] It would be better if the todo list are loaded or displayed in the browser on page load. However, currently we are not displaying the todos on page load as we have in the screenshot below

![Screenshot from 2022-08-11 15-44-37](https://user-images.githubusercontent.com/46388113/184174804-312ae727-330c-416d-b93b-ed2514605974.png)

This part of the project requirement has not been met
![Screenshot from 2022-08-11 15-45-29](https://user-images.githubusercontent.com/46388113/184175043-85e39190-2975-4a73-bbe5-56ad84bdfb43.png)

Reply:
@addod19 This is webpack, I think the file should be opened from the DIST (distribute) HTML file (output) and not the source (src file) HTML.

Reply:
All Done

Reply:
Well resolved 👍🏽